src/Angular.js

```diff
@@ -807,6 +855,7 @@ function arrayRemove(array, value) {
  *                   Can be any type, including primitives, `null`, and `undefined`.
  * @param {(Object|Array)=} destination Destination into which the source is copied. If
  *     provided, must be of the same type as `source`.
+ * @param {Number=} maxDepth All properties of the source will be copied until reaching the max depth.
```

Comment:
We discussed this internally and we decided that it is better not to make this parameter public at this point.
So, please remove any mentions from the `copy` docs.